docs/GettingStarted.md

```diff
@@ -22,27 +25,27 @@ By default, when the cluster is started, the data store is bootstrapped and when
+* **Mutate, transact on data in Spark**: You can use SQL to insert, update, delete data in tables as one would expect. Extensions to Spark’s context are also provided so you can mutate data in your Spark programs. Tables defined in SnappyData are automatically visible as DataFrames. By eliminating the need to store data separately in a data store and then cache it in Spark for compute processing, SnappyData simplifies system architecture and reduces the overall cost of ownership while simultaneously offering much better performance.
 
-* **Optimizations - Indexing**: You can index your RowStore and the GemFire SQL optimizer, which automatically uses in-memory indexes when available.
+* **Optimizations - Indexing**: From version 1.0, you can add indexes to your RowStore and the query optimizer automatically uses in-memory indexes when available, to provide better performance.
 
-* **Optimizations - collocation**: SnappyData implements several optimizations to improve data locality and avoid shuffling data for queries on partitioned data sets. All related data can be collocated using declarative custom partitioning strategies (for example, common shared business key). Reference data tables can be modeled as replicated tables when tables cannot share a common key. Replicas are always consistent.
+* **[Optimizations - colocation](howto.md#howto-collacatedJoin)**: SnappyData implements several optimizations to improve data locality and avoid shuffling data for queries on partitioned data sets. All related data can be colocated using declarative custom partitioning strategies (for example, common shared business key). Reference data tables can be modeled as replicated tables when tables cannot share a common key. Replicas are always consistent.
 
 * **High availability not just Fault tolerance**: Data can be instantly replicated (one at a time or batch at a time) to other nodes in the cluster. It is deeply integrated with a membership-based distributed system to detect and handle failures, instantaneously providing applications continuous HA.
 
-* **Durability and recovery:** Data can also be managed on disk and automatically recovered. Utilities for backup and restore are bundled.
+* **Durability and recovery:** Tables can be configured to be persisted to disk and recovered upon startup. Utilities for backup, restore and import/export are provided with the system.
+
+* **[Interactive analytics using Synopsis Data Engine (SDE)](aqp.md)**: Multiple synopses techniques are introduced through data structures like count-min-sketch and stratified sampling to dramatically reduce the in-memory space requirements and provide true interactive speeds for analytic queries. These structures can be created and managed by developers with little to no statistical background and can be completely transparent to the SQL developer running queries. Error estimators are also integrated with simple mechanisms to get to the errors through built-in SQL functions. 
 
```

Comment:
can be completely transparent => are completely transparent

add at the end: ... SQL functions **or SparkSession API extensions**.